src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBodyMapper.java

```diff
@@ -103,4 +102,20 @@ public DocAssemblyTemplateBody from(CCDCase ccdCase, UserDetails userDetails) {
                 ccdCase.getExpertReportPermissionPartyGivenToDefendant() == YES)
             .build();
     }
+
+    private HearingCourt getHearingCourt(CCDCase ccdCase) {
```

Comment:
This method is duplicated in two different classes, please extract it to some common place and add unit tests.